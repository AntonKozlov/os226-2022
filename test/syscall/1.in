syscalltest hello
retcode

syscalltest people!
retcode
