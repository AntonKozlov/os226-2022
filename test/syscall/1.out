hello
5
people!
7
